<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1452</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QFleet</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc/icons.qrc">
    <normaloff>:/dfc.jpeg</normaloff>:/dfc.jpeg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,0,3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="selectedShipLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="selectedShipNameLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="shipMenuButton">
            <property name="text">
             <string>Ship Menu</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="contentBox">
        <item>
         <layout class="QVBoxLayout" name="fleetBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1452</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>List</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave_Compressed"/>
    <addaction name="actionLoad_compressed"/>
   </widget>
   <widget class="QMenu" name="menuPrint">
    <property name="title">
     <string>Print</string>
    </property>
    <addaction name="actionFleet_List"/>
    <addaction name="actionCompact_Fleet_List"/>
    <addaction name="actionStrategy_Cards"/>
    <addaction name="actionSimple_List"/>
    <addaction name="actionProfile_Cards"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPrint"/>
  </widget>
  <action name="actionFleet_List">
   <property name="text">
    <string>Tradtional Fleet List</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionLoad_Ships">
   <property name="text">
    <string>Load Ships</string>
   </property>
  </action>
  <action name="actionLoad_Ships_and_Launch">
   <property name="text">
    <string>Load Ships and Launch</string>
   </property>
  </action>
  <action name="actionLoad_Launch">
   <property name="text">
    <string>Load Launch</string>
   </property>
  </action>
  <action name="actionStrategy_Cards">
   <property name="text">
    <string>Strategy Cards</string>
   </property>
  </action>
  <action name="actionSimple_List">
   <property name="text">
    <string>Simple List</string>
   </property>
  </action>
  <action name="actionCompact_Fleet_List">
   <property name="text">
    <string>Compact Fleet List</string>
   </property>
  </action>
  <action name="actionSave_Compressed">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad_compressed">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionProfile_Cards">
   <property name="text">
    <string>Profile Cards</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rsc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
